app/controllers/TpsController.scala

```diff
@@ -65,6 +65,15 @@ class TpsController @Inject() (actions: Actions,
     Future.successful(Ok(toJson(TpsId.fresh)))
   }
 
+  def getPaymentItem(id: PaymentItemId): Action[AnyContent] = actions.strideAuthenticateAction().async {
+    Logger.debug(s"getPaymentItem ${id.value}")
+
+    tpsRepo.findPaymentItem(id).map {
+      case Some(x) => Ok(toJson(x))
```

Comment:
At some point the internal model of PaymentItem might evolve, for example new fields could be added/removed/renamed. This change will compile and pass tests. However it might introduce problem on caller site.

What we do is we use dedicated layer of model/case classes to encapsulate such API.
See for example `pay-api`. It never exposes it's Journey, but only projections of it ( RJourney - for reconciliation for reconciliation, Payment - for Amls)
 
Please consider above and implement projection classes for this API.

Reply:
As discussed, we only need TaxType so added this as a projection in the endpoint and returned just the taxType